examples/ConsumptionSaving/example_ConsRiskyContribModel.py

```diff
@@ -0,0 +1,364 @@
+# %%
+"""
+Example implementations of HARK.ConsumptionSaving.ConsPortfolioModel
+"""
+from HARK.ConsumptionSaving.ConsRiskyContribModel import (
+    RiskyContribConsumerType,
+    init_risky_contrib,
+)
+from time import time
+import numpy as np
+
+# %% Define a plotting function
+def plotFuncs3D(functions, bottom, top, N=300, titles=None, ax_labs=None):
```

Comment:
Most of this example is these plotting functions.

in utilities.py there is a plot_funcs command. maybe it would be appropriate to move these there?

In any case, please respect the Python style guide of `snake_case` for function names

Reply:
Addressed in https://github.com/econ-ark/HARK/pull/832/commits/453140a9b4648e3885524e44e23a5e627c398744

Some tool like the plotting functions defined there might be useful in `utilities.py`. However, the ones I've created here are specific to this particular model, and I don't think they'd be too useful in other contexts without substantial changes.